build_tools/cmake/iree_benchmark_suite.cmake

```diff
@@ -237,10 +237,51 @@ function(iree_benchmark_suite)
         add_dependencies(iree-benchmark-suites "${_TRANSLATION_TARGET_NAME}")
       endif(NOT TARGET "${_TRANSLATION_TARGET_NAME}")
 
+      set(_COMPILATION_STATS_TRANSLATION_TARGET_NAME
+        "${_TRANSLATION_TARGET_NAME}-compilation-stats"
+      )
+      set(_COMPILATION_STATS_VMFB_FILE
+        "${_VMFB_ARTIFACTS_DIR}/${_MODULE_SOURCE_BASENAME_WITH_HASH}-compilation-stats.vmfb"
+      )
```

Comment:
How long are these names? Want to abbreviate to `compile-stats` or `stats`?

Reply:
`compile-stats` looks good to me, since we already shorten `statistics` to `stats`.